gradle.properties

```diff
@@ -15,8 +15,8 @@ forge_version=44.0.41
 //forge_ats_enabled=true
 
 # Fabric
-fabric_version=0.70.0+1.19.3
-fabric_loader_version=0.14.8
+fabric_version=0.76.0+1.19.4
+fabric_loader_version=0.14.17
```

Comment:
Same as comment as above

Reply:
Removed this change, looks like it works properly just by adjusting the fabric deps